app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,25 +3,58 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_VALUES = {
   title: "",
   description: "",
-  assignedContactId: "",
-  tag: "",
+  assignedContact: [],
```

Comment:
Same as last comment.

Reply:
@LiyakathMohammed https://www.loom.com/share/99d16c33e41041ee87db966ece9d27a5

Reply:
Fixed it by assigning both assignedTo and tag to null.